src/Platforms/AbstractPlatform.php

```diff
@@ -337,7 +337,7 @@ abstract public function getBlobTypeDeclarationSQL(array $column);
     /**
      * Gets the name of the platform.
      *
-     * @return string
+     * @return Family::*
```

Comment:
This return type restriction makes impossible to implement any non-standard platform